frontend/src/components/Pages/RobotPage/RobotArmMovement.tsx

```diff
@@ -0,0 +1,81 @@
+import { Autocomplete, AutocompleteChanges, Button, Card, Dialog, Typography, Icon } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { useState, useEffect } from 'react'
+import { BackendAPICaller } from 'api/ApiCaller'
+import { Robot } from 'models/Robot'
+
+const StyledDialog = styled(Card)`
+    display: flex;
+    padding: 1rem;
+    width: 600px;
+    right: 175px;
+`
+const StyledAutoComplete = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+`
+
+const StyledButtons = styled.div`
+    display: flex;
+    gap: 8px;
+    justify-content: flex-end;
+`
+
+interface RobotProps {
+    robot: Robot
+}
+
+export function BatteryPose({ robot }: RobotProps) {
+    const [feedback, setFeedback] = useState('')
+
+    const onClickBatteryPose = () => {
+        BackendAPICaller.postArmPosition(robot.id, 'battery_change') //VALID_ARM_POSES = '["transport","battery_change","lookout"]'
+            .then(() => {
+                setFeedback(`Moving arm to Battery Pose for robot  ${robot.name}`)
+                setTimeout(() => {
+                    setFeedback('')
+                }, 10000) // Clear feedback after 10 seconds
```

Comment:
```suggestion
                }, 10000)
```

Reply:
With a good name for a configurable variable there will be no need for the comment